app/static/css/style.css

```diff
@@ -184,7 +184,7 @@ footer{
 	color: var(--grey-mid);
 }
 
-.row [class^=col-]:not(.col-12){
+.row:not(.no-space) [class^=col-]:not(.col-12) {
```

Comment:
Is this related to this PR? I don't see `.no-space` used anywhere.

Reply:
It was